### DISCUSSION TITLE
Indexing size and time

### DISCUSSION THREAD

**Discussion by flowingblaze** [nicobrauchtgit/obsidian-Smart2Brain#129] [Q&A]
Does anyone know how much space indexing will take. Does it depend on your count of notes or vault size? Or both?

**Comment by flowingblaze** [comment]
It says it will take around 4k to 5k minutes to index

**Comment by nicobrauchtgit** [comment]
Hi @flowingblaze, the space the indexing will take up mainly depends on the vault size in general and the way you structure your vault with headings, subheadings, paragraphs and lists in particular but not really on the note count. 
However the time it takes to index your vault mainly depends on the size of your vault and the choice of embedding model. In case of a local embedding model the time also depends on the specs of the machine they are run on. Furthermore, the estimated time is dependant on the indexing duration of the last 5 documents and thus may vary. Hope this helps! 😊
